src/locales/it/translation.json

```
@@ -328,129 +328,129 @@
-                        "XX is the arbitrary code that the supplier used to identify the groups of cards.",
-                        "Y is the current number of the pack out of total packs for code XX.",
-                        "Z is the total number of packs for value XX."
+                        "Questo è il titolo sulla carta grigia presente in ogni mazzo di carte impacchettato. E' il modo in cui mi riferisco a ciascun pacchetto.",
+                        "XX è il codice arbitrario utilizzato dal fornitore per identificare i gruppi di carte.",
+                        "Y è il numero corrente del pacchetto sul totale dei pacchetti per il codice XX.",
+                        "Z è il numero totale di pacchetti per il valore XX."
                     ]
                 },
                 {
                     "title": "$t(common.expansions)",
                     "details": [
-                        "These are the expansions that the cards belong to in this pack."
+                        "Queste sono le espansioni a cui appartengono le carte di questo pacchetto."
                     ]
                 },
                 {
-                    "title": "Example Type Cards: X (Y/Z) OR X/Z",
+                    "title": "Esempio Tipo di Carte: X (Y/Z) oppure X/Z",
                     "details": [
-                        "Example Type is the type of card, as referred to in Go On Board's \"Components\" lists.",
-                        "X is how many of the cards are found in this pack.",
-                        "Y is the current running total of this type of card across all packs. If excluded, all cards of this type were included in this pack",
-                        "Z is the total number of these cards you should have."
+                        "Esempio Tipo è il tipo di carta, come indicato negli elenchi \"Componenti\" della Go On Board.",
+                        "X è il numero di carte presenti in questo pacchetto.",
+                        "Y è il totale corrente di questo tipo di carta in tutti i pacchetti. Se viene omesso, tutte le carte di questo tipo sono incluse in questo pacchetto",
+                        "Z è il numero totale di queste carte che dovresti avere."
                     ]
                 },
                 {
                     "title": "(AP)",
                     "details": [
-                        "This is a disambiguation for Adventure Pack cards. These are distinguished by the star symbol in the top center of the card face."
+                        "Questa è una soluzione per chiarire quali siano le carte dell'Espansione \"Avventure\". Queste si distinguono per il simbolo con un piccola stella in alto al centro del lato anteriore della carta."
                     ]
                 }
             ]
         },
-        "card": {
-            "numInPack": "# in this pack",
-            "numSoFar": "# so far",
-            "numTotal": "# total"
+         "card": {
```

```suggestion
        "card": {
```

This accidentally had an additional space included, which will prevent the json from being valid and working properly.